test/transcoding/SPV_INTEL_joint_matrix/joint_matrix_bfloat16.ll

```diff
@@ -0,0 +1,201 @@
+; RUN: llvm-as < %s -o %t.bc
+; RUN: llvm-spirv %t.bc --spirv-ext=+SPV_INTEL_bfloat16_conversion,+SPV_INTEL_joint_matrix -o %t.spv
```

Comment:
Could you add a `llvm-spirv -s` check to see what was changed after regularization?